readme.md

```diff
@@ -660,6 +660,30 @@ As of version `5.0.0` it uses source maps to report coverage for your actual cod
 
 ## FAQ
 
+### How do I test HTTP endpoints?
+
+AVA doesn't have an official assertion library for endpoints, but a great option is [`supertest-as-promised`](https://www.npmjs.com/package/supertest-as-promised).
+Assuming you are using express, you might set up a server for each test:
+```
+let app;
+test.beforeEach(() => {
```

Comment:
good to know, thanks! a `before` would probably suffice, but I wasn't sure if it was a "best practice" since someone could mutate `app` within a test.

if you don't mind, could you explain why `context` is needed & a `let` doesn't suffice? I'm guessing it's because the `let` won't carry through to all the child processes?